package.json

```diff
@@ -84,7 +84,7 @@
     "@react-native-community/clipboard": "git+https://github.com/brunobar79/clipboard#01d1c13",
     "@react-native-community/masked-view": "0.1.10",
     "@react-native-community/netinfo": "5.9.7",
-    "@react-native-community/viewpager": "5.0.5",
```

Comment:
CI error "Patch file found for package viewpager which is not present at node_modules/@react-native-community/viewpager". Sounds like you just need to delete this leftover file